guide/src/generate_text_using_the_vertex_ai_gemini_api.md

```diff
@@ -123,4 +137,9 @@ ______________________________________________________________________
 {{#include ../samples/src/gemini.rs:prompt-and-image}}
 ```
 
+Note: These code samples return `crate::Result<()>` for the sake of brevity. You
+will need to change the return type to
+`std::result::Result<(), Box<dyn std::error::Error>>` since your project
+structure is likely different from that of the samples.
+
```

Comment:
Hmm... projects may use `anyhow::Result<()>` or the `Box` thing ... or their own error that wraps our error as a source.

Maybe we should change all the code to use `anyhow::Result<()>`